benchmarks/01_trivial/05_packed_load_store.cpp

```diff
@@ -0,0 +1,191 @@
+#include <benchmark/benchmark.h>
+#include <bitset>
+#include <cstdint>
+#include <stdio.h>
+
+#include "../common.h"
+#include "05_packed_load_store_ispc.h"
+
+static Docs docs("Check packed_load_active/packed_store_active implmentation of stdlib functions:\n"
+                 "[int32, uint32, int64, uint64] x [all, even, four, greater] versions.\n");
```

Comment:
Looking at `[all, even, four, greater]` I think they are good for experiments, but doesn't worth having all of them running regularly - too much noise, which doesn't have enough information.
I suggest to keep `all` and the version, which stores/loads a single value - the reasoning is that we'd need to measure two extremes. To have a single value version I suggest comparing `programIndex == idx`, where `idx` is function argument - to prevent dead code elimination from removing parts of intrinsic.
Running `all` will have emphasis on doing more memory operations, while single value version will have more emphasis on overhead, like using `popcnt()` to count the active lanes.

Reply:
Done